frontend/app/src/components/Header/NavMenuMobile/index.js

```diff
@@ -0,0 +1,59 @@
+import React from "react";
```

Comment:
It would be better to have this one as responsive component with NavMenu

Reply:
@pholm is the author of this, he knows more